src/panels/gitConfigPanel.ts

```diff
@@ -14,57 +13,60 @@ export function activateGitConfigWebview(context: vscode.ExtensionContext) {
 class GitConfigProvider implements vscode.WebviewViewProvider {
     public static readonly viewType = "gitConfig";
     private _disposables: vscode.Disposable[] = [];
-    private currentRootPath: string;
+    private currentRootPathAndLabel: [string, string];
```

Comment:
Can we rewrite this with an interface so that we don't have to do [0] and [1] through the code?